src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/cluster.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="thibault" id="cluster">
+        <createTable tableName="CLUSTER">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="clusterName" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="googleId" type="BINARY(16)">
```

Comment:
Yeah, anything that you "where" on (i.e. the `get` functions in your `FooComponent`) should be indexed. so google id and the tuple (googleProject, clusterName) too